.vib/vib-scheduled-verify.json

```diff
@@ -23,6 +23,23 @@
           "action_id": "container-image-lint"
         }
       ]
+    },
+    "verify": {
+      "context": {
```

Comment:
Do you need this section here?

Reply:
Yes, you are right. I need some additional parameterization

Reply:
I doubt if we need the `target_platform` since we are not deploying...

Reply:
`target_platform` is not required, Trivy analyse directly the image